src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -0,0 +1,289 @@
+const simpleGit = require("simple-git");
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+const {
+  createAmplifyBranches,
+  startReleaseJob,
+  createAmplifyApp,
+  readBuildSpec,
+} = require("./amplifyUtils");
+const BRANCH_NAME = "chore-amplify-migration-change-permalinks";
+const ORGANIZATION_NAME = "isomerpages";
+
+interface AmplifyAppInfo {
+  appId: string;
+  repoName: string;
+  name: string;
+  repoPath: string;
+}
+
+const listOfRepos: [string, string][] = [
+  ["repo-name", "Human Friendly Name"], // Modify this line when running the script
+];
```

Comment:
Could we change this to use command line arguments instead? might be easier than having to replace within the code

Reply:
sure, 17c94d5